README.md

```diff
@@ -4,17 +4,40 @@ League Uri Parser
 [![Build Status](https://img.shields.io/travis/thephpleague/uri-parser/master.svg?style=flat-square)](https://travis-ci.org/thephpleague/uri-parser)
 [![Latest Version](https://img.shields.io/github/release/thephpleague/uri-parser.svg?style=flat-square)](https://github.com/thephpleague/uri-parser/releases)
 
-This package contains a userland PHP uri parser compliant with [RFC 3986](http://tools.ietf.org/html/rfc3986).
+This package contains a userland PHP uri parser and builder compliant with:
```

Comment:
URI?

Reply:
yes URI because RFC3986 talks about URI and URLs are bit a subset of URIs.

Reply:
I mean, in uppercase :) ?

Reply:
ah yes ... let me fix that :smile: